docs-ref-services/legacy/ai-formrecognizer-readme.md

```diff
@@ -1,14 +1,13 @@
 ---
 title: Azure Form Recognizer client library for Java
+description: Learn how to use machine learning to recognize text and table data from form documents with Azure Cognitive Services Form Recognizer.
 keywords: Azure, java, SDK, API, azure-ai-formrecognizer, formrecognizer
-author: ramya-rao-a
-ms.author: ramyar
+author: samvaity
+ms.author: savaity
 ms.date: 11/12/2020
 ms.topic: reference
-ms.prod: azure
-ms.technology: azure
 ms.devlang: java
-ms.service: formrecognizer
+ms.service: azure-java
```

Comment:
Should this be ms.service : applied-ai-services as mentioned here https://github.com/Azure/azure-docs-sdk-java/pull/2209/files#diff-e27560848ba6186c6b8458e3c99f880a6d02482f07a95d547016a957f9d765e4L11

Reply:
@samvaity I have made this change. Let me know if you would like any other changes. Thank you :)